Amp.js

```diff
@@ -1,69 +1,100 @@
-'use strict';
-
+const axios = require('axios');
 const Session = require('./Session');
-const Version = require('./Version');
+const Utils = require('./Utils');
 
+const utils = new Utils();
 /**
+ *
  * Amp
  * Constructs Amp instance with passed configuration.
- *
  * @constructor
- * @param {Object} options
- *
- * Options:
- *  key - project key
- *  domain - domain of Amp server
- *  apiPath - path of API
- *  sessionTTL - TTL for session
- *  timeout - TTL for requests
+ * @param  {string} key - project key
+ * @param  {Array} ampAgents - list of ampAgents
+ * @param  {int} timeOutMilliseconds - request timeOut in milliSeconds default 10 seconds
+ * @param  {int} sessionLifeTimeSeconds - session life time in Seconds default 30 minutes
+ * @param  {boolean} dontUseTokens - flag to indicate either generate amptoken automatically or use the custom token.
  *
  */
-module.exports = class Amp {
-  constructor(options = {}) {
-    this.key = options.key;
-    if (!this.key) throw new Error('Project Key Needed!');
+module.exports  = class AmpV2 {
```

Comment:
Why are you changing the shared API?

Reply:
This was a mistake. I forgot to make the change to Amp. Fixed right now.